ui/components/multichain/network-list-menu/network-list-menu.tsx

```diff
@@ -195,12 +196,15 @@ export const NetworkListMenu = ({ onClose }: { onClose: () => void }) => {
     () =>
       Object.entries(multichainNetworks).reduce(
         ([nonTestnetsList, testnetsList], [id, network]) => {
+          let isTest = false;
           const chainId = network.isEvm
             ? convertCaipToHexChainId(id as CaipChainId)
             : id;
-          // This is type casted to string since chainId could be
-          // Hex or CaipChainId.
-          const isTest = (TEST_CHAINS as string[]).includes(chainId);
+          if (network.isEvm) {
+            isTest = (TEST_CHAINS as string[]).includes(chainId);
+          } else {
+            isTest = NON_EVM_TESTNET_IDS.includes(chainId as CaipChainId);
+          }
```

Comment:
Not sure if `id` and `network.chainId` are always the same (but I assume yet?), we could do this instead:

```ts
          let chainId = id;
          let isTest = false;

          if (network.isEvm) {
            // We keep using raw chain ID for EVM.
            chainId = convertCaipToHexChainId(network.chainId)
            isTest = TEST_CHAINS.includes(chainId as Hex);
          } else {
            isTest = NON_EVM_TESTNET_IDS.includes(network.chainId);
          }
```

Reply:
https://github.com/MetaMask/metamask-extension/pull/31702/commits/951c0efd729e180d881e0c51ffdc75f9f75a2960